qa/build.xml

```diff
@@ -29,11 +29,12 @@
   <!-- you should not need to configure anything below here -->
   <!--
         This is a redhat vpn server. for non-vpn use, mirror it locally or copy the drivers by hand.
```

Comment:
Just noticed that the old server www.qa.jboss.com was also behind the VPN. In fact, for non-VPN use, it is suggested to copy all needed drivers manually.